dvc/api/__init__.py

```diff
@@ -1,9 +1,15 @@
-from .data import (  # noqa, pylint: disable=redefined-builtin
-    get_url,
-    open,
-    read,
-)
+from dvc.fs.dvc import _DvcFileSystem as DvcFileSystem
```

Comment:
Does it make sense to still name it with `_` prefix if it's now public?

Reply:
The problem is that we also have `dvc.fs.DvcFileSystem`, which is our own wrapper.

Reply:
I see.
Perhaps I am just being pedantic 😅 but looks like our own wrapper should be the one prefixed with `_`.

Doesn't really matter, just found it confusing

Reply:
Yeah, but we'll need to work with that prefixed filesystem more than this one, import that, etc. So it may be confusing for us too.

I don't like it either, because we are re-exporting here and their original location will be seen in REPLs.
```python
>>> from dvc.api import DvcFileSystem
>>> DvcFileSystem
dvc.fs.dvc._DvcFileSystem
>>> DvcFileSystem()
<dvc.fs.dvc._DvcFileSystem at 0x7f6548ccd310>
```

The solution is to either subclass `_DvcFileSystem` in `dvc.api` to `DvcFileSystem` which is ugly, or rename the wrapper one to something else, which is confusing.